filters/scanner_test.go

```diff
@@ -271,6 +271,84 @@ func TestScanner(t *testing.T) {
 				{pos: 23, token: tokenEOF},
 			},
 		},
+		{
+			name:  "labelsHasQuota",
+			input: "labels.containerd.io/uncompressed==3522",
+			expected: []tokenResult{
+				{pos: 0, token: tokenField, text: "labels"},
+				{pos: 6, token: tokenSeparator, text: "."},
+				{pos: 7, token: tokenField, text: "containerd"},
+				{pos: 17, token: tokenSeparator, text: "."},
+				{pos: 18, token: tokenField, text: "io/uncompressed"},
+				{pos: 33, token: tokenOperator, text: "=="},
+				{pos: 35, token: tokenValue, text: "3522"},
+				{pos: 39, token: tokenEOF},
+			},
+		},
+		{
+			name:  "labelsHasQuotaNoValue",
+			input: "labels.containerd.io/uncompressed",
+			expected: []tokenResult{
+				{pos: 0, token: tokenField, text: "labels"},
+				{pos: 6, token: tokenSeparator, text: "."},
+				{pos: 7, token: tokenField, text: "containerd"},
+				{pos: 17, token: tokenSeparator, text: "."},
+				{pos: 18, token: tokenField, text: "io/uncompressed"},
+				{pos: 33, token: tokenEOF},
+			},
+		},
+		{
+			name:  "labelsHasQuotaIntheEnd",
+			input: "labels.containerd.io/",
+			expected: []tokenResult{
+				{pos: 0, token: tokenField, text: "labels"},
+				{pos: 6, token: tokenSeparator, text: "."},
+				{pos: 7, token: tokenField, text: "containerd"},
+				{pos: 17, token: tokenSeparator, text: "."},
+				{pos: 18, token: tokenField, text: "io/"},
+				{pos: 21, token: tokenEOF},
+			},
+		},
+		{
+			name:  "labelsHasQuotaAfterField",
+			input: "labels.containerd.io/test/",
+			expected: []tokenResult{
+				{pos: 0, token: tokenField, text: "labels"},
+				{pos: 6, token: tokenSeparator, text: "."},
+				{pos: 7, token: tokenField, text: "containerd"},
+				{pos: 17, token: tokenSeparator, text: "."},
+				{pos: 18, token: tokenField, text: "io"},
+				{pos: 20, token: tokenQuoted, text: "/test/"},
```

Comment:
This doesn't need to show up as a quoted value, the first should just be `io/test/`

The grammar is 
```
selectors := selector ("," selector)*
selector  := fieldpath (operator value)
fieldpath := field ('.' field)*
field     := quoted | [A-Za-z] [A-Za-z0-9_]+
operator  := "==" | "!=" | "~="
value     := quoted | [^\s,]+
quoted    := <go string syntax>
```
Disallowing a quoted value in the middle of a field.

Rather we just want it updated like this
```diff
- field     := quoted | [A-Za-z] [A-Za-z0-9_]+
+ field     := quoted | [A-Za-z] [A-Za-z0-9_/]+
```